app/models/info_request.rb

```diff
@@ -39,7 +39,8 @@ class InfoRequest < ActiveRecord::Base
   validates_presence_of :title, :message => N_("Please enter a summary of your request")
   validates_format_of :title, :with => /[[:alpha:]]/,
     :message => N_("Please write a summary with some text in it"),
-    :if => Proc.new { |info_request| !info_request.title.nil? && !info_request.title.empty? }
+    :unless => Proc.new { |info_request| info_request.title.blank? }
+  validate :title, :max_length => 200, :message => _('Please keep the summary short, like in the subject of an email. You can use a phrase, rather than a full sentence.')
```

Comment:
not sure whether this will break any older requests which weren't originally subject to the length restriction, if so I should add an `:on => :create` so that it only applies to new requests